app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :bookings
+  has_many :doctors, through: :bookings
```

Comment:
- I understand, in the future, a doctor could add notes for the user/patient here but, at this point, it looks like this app is only for bookings, so, why add a relation between doctor and user? I consider it good to add this relation if you have the plan to add some info regarding these two but if you don't have it. Maybe you consider no relating these tables (the same for doctor.rb) 🐦

Reply:
Done, Thanks